README.md

```diff
@@ -2,74 +2,12 @@
 Bowling Challenge
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+* All the logic is working.
+* But the CLI is not.
+* So currently the programme is set for every frame to be a strike.
 
-## The Task
+Next steps:
+============
```

Comment:
If you had more time, maybe you could add some more instructions or explanation of how your code fits together. Your class separation is fantastic, so maybe you could include a brief overview of what each class is responsible for here in your readme